lib/notifications-status-view.js

```diff
@@ -0,0 +1,76 @@
+const { CompositeDisposable, Disposable } = require('atom');
+const moment = require('moment');
+
+module.exports = class NotificationsStatusView {
+
+    constructor(statusBar, duplicateTimeDelay) {
+      this.count = 0;
+      this.duplicateTimeDelay = duplicateTimeDelay;
+      this.subscriptions = new CompositeDisposable;
+      this.render();
+      this.tile = statusBar.addRightTile({
+        item: this.element,
+        priority: 100
+      });
+      this.subscriptions.add(new Disposable(() => { this.tile.destroy(); }));
+      // TODO: uncomment next line when atom/atom#16074 is released
+      // this.subscriptions.add(atom.notifications.onDidClearNotifications(() => this.clear()));
+    }
+
+    render() {
+      this.number = document.createElement('div');
+      this.number.textContent = this.count;
+      this.number.addEventListener('animationend', (e) => {
```

Comment:
I'm worried that this will be too distracting.  Did you think the number changing was too subtle to notice without an animation?

Reply:
Ya, sometimes I like turning the notification popups off when I am trying to concentrate and that seemed like a good way not give it a little pop.

Also that way people can easily turn it off or specify a different animation by adding the following to their `styles.less`

```css
.notifications-count .new-notification {
  animation: none;
}
```

Reply:
this could allow other packages like [`notifications-plus-confetti`](https://atom.io/packages/notifications-plus-confetti)

![notifications-plus-confetti](https://i.github-camo.com/fd3e96e1d555406b26812ccaa3a10cfee72f5f27/68747470733a2f2f757365722d696d616765732e67697468756275736572636f6e74656e742e636f6d2f39373939342f32393233383431372d62356134333332342d376566612d313165372d383362362d3065636636343666363736362e676966)